mapreduce.go

```diff
@@ -218,7 +218,7 @@ func (h urlHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, "Internal server error", http.StatusInternalServerError)
 		return
 	}
-	log := appwrap.NewAppengineLogging(c)
```

Comment:
this one should be fine to leave.  we wrap the handlers appropriately everywhere afaik.  and if we don't, we should.